proj/src/main/webapp/test/chat.test.js

```diff
@@ -1,6 +1,8 @@
 // Chai is a commonly used library for creating unit test suites. It is easily extended with plugins.
 const chai = require('chai');
 const assert = chai.assert;
+const expect = chai.expect;
```

Comment:
it looks like this isn't being used. Can it be removed?

Reply:
yup this was suppose to be in the testing branch but i will delete the unused code here